sigstore-java/src/main/java/dev/sigstore/tuf/Updater.java

```diff
@@ -139,6 +141,8 @@ Root updateRoot()
     throwIfExpired(expires);
     // 5.3.11) If the timestamp and / or snapshot keys have been rotated, then delete the trusted
     // timestamp and snapshot metadata files.
+    // TODO(patrick): have Asra look at this again. What if an old key is remove but there are no
```

Comment:
@asraa

Reply:
I think you would fail verification of the snapshot/timestamp when updating those -- if there's no keys, then it can't hit the snapshot/timestamp signing threshold
i don't think that needs to be covered in this flow

Reply:
Thanks for taking a look @asraa .  I think what wasn't clear to me from the spec is what constitutes 'rotating keys'.   I'm taking from your comment that I should leave as is, and 'rotating' is equivalent to adding one or more new keys regardless of whether old keys are kept or dropped.